docs/build_and_deploy.md

```diff
@@ -0,0 +1,15 @@
+## Build and Deploy
```

Comment:
This is great but build and deploy should probably be left for orchestration and config related to building and deploying (like [CCCDS build and deploy doc](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/docs/build_and_deploy.md)).

This would be betters as its on doc. I would suggest just renaming this `docs/monitoring.md` (and amend title)

Reply:
👍 Have renamed to docs/monitoring.md and amended title